cli/tests/test_revset_output.rs

```diff
@@ -770,7 +767,7 @@ fn test_revset_committer_date_with_time_zone() {
 
     let mut log_commits_before_and_after = |committer_date: &str, now: &str, tz: &str| {
         test_env.add_env_var("TZ", tz);
-        let config = format!("debug.commit-timestamp={now}");
+        let config = format!(r#"debug.commit-timestamp="{now}""#);
```

Comment:
tbh I think reverting just this one may be fine but meh they're both ugly given the interpolation
```suggestion
        let config = format!("debug.commit-timestamp=\"{now}\"");
```